systemtests/src/test/java/io/enmasse/systemtest/iot/isolated/project/ManagedTest.java

```diff
@@ -36,29 +49,66 @@
 import io.fabric8.kubernetes.client.dsl.MixedOperation;
 import io.fabric8.kubernetes.client.dsl.Resource;
 
-@Tag(TestTag.SHARED_IOT)
-public class ManagedTest extends AbstractIoTProjectTestBase  {
+public class ManagedTest extends TestBase implements ITestIoTIsolated {
 
     private static final Logger log = CustomLogger.getLogger();
 
     private MixedOperation<IoTProject, IoTProjectList, DoneableIoTProject, Resource<IoTProject, DoneableIoTProject>> client;
     private MixedOperation<Address, AddressList, DoneableAddress, Resource<Address, DoneableAddress>> addressClient;
     private MixedOperation<AddressSpace, AddressSpaceList, DoneableAddressSpace, Resource<AddressSpace, DoneableAddressSpace>> addressSpaceClient;
     private MixedOperation<User, UserList, DoneableUser, Resource<User, DoneableUser>> userClient;
+    protected DeviceRegistryClient registryClient;
+    protected CredentialsRegistryClient credentialsClient;
 
     @BeforeEach
-    public void initClients () {
-        this.client = kubernetes.getIoTProjectClient(this.iotProjectNamespace);
-        this.addressClient = kubernetes.getAddressClient(this.iotProjectNamespace);
-        this.addressSpaceClient = kubernetes.getAddressSpaceClient(this.iotProjectNamespace);
-        this.userClient = kubernetes.getUserClient(this.iotProjectNamespace);
+    public void initClients () throws Exception {
+        CertBundle certBundle = CertificateUtils.createCertBundle();
+        IoTConfig iotConfig = new IoTConfigBuilder()
+                .withNewMetadata()
+                .withName("default")
+                .endMetadata()
+                .withNewSpec()
+                .withNewServices()
+                .withDeviceRegistry(DefaultDeviceRegistry.newInfinispanBased())
+                .endServices()
+                .withNewAdapters()
+                .withNewMqtt()
+                .withNewEndpoint()
+                .withNewKeyCertificateStrategy()
+                .withCertificate(ByteBuffer.wrap(certBundle.getCert().getBytes()))
+                .withKey(ByteBuffer.wrap(certBundle.getKey().getBytes()))
+                .endKeyCertificateStrategy()
+                .endEndpoint()
+                .endMqtt()
+                .endAdapters()
+                .endSpec()
+                .build();
+
+        isolatedIoTManager.createIoTConfig(iotConfig);
+
+        final Endpoint deviceRegistryEndpoint = kubernetes.getExternalEndpoint("device-registry");
+        registryClient = new DeviceRegistryClient(kubernetes, deviceRegistryEndpoint);
+        credentialsClient = new CredentialsRegistryClient(kubernetes, deviceRegistryEndpoint);
+        this.client = kubernetes.getIoTProjectClient(this.IOT_PROJECT_NAMESPACE);
```

Comment:
`IOT_PROJECT_NAMESPACE` is static, and so it should not be referenced by `this.`. Some for the others.